### PR TITLE
Update output messages

### PR DESCRIPTION
**Pull request by JoelPasvolsky**
@bellert some minor changes to the output notices for user clarity

### CODE REVIEW

File: dwave_structural_imbalance_demo/interfaces.py [hunk 1]
```diff
@@ -133,17 +133,17 @@ def solve_structural_imbalance(self, subregion='Global', year=None):
         if self._qpu:
             try:
                 imbalance, bicoloring = dnx.structural_imbalance(G, self._embedding_composite)
-                print("Ran on the QPU using EmbeddingComposite")
+                print("Ran on the QPU using dimod's EmbeddingComposite")
```
Comment (bellert) [inline dwave_structural_imbalance_demo/interfaces.py]: The embedding composite is part of `dwave-system`

File: dwave_structural_imbalance_demo/interfaces.py [hunk 2]
```diff
@@ -133,17 +133,17 @@ def solve_structural_imbalance(self, subregion='Global', year=None):
...
             except ValueError:
                 imbalance, bicoloring = dnx.structural_imbalance(G, self._qbsolv, solver=self._embedding_composite)
-                print("Ran on the QPU using QBSolv w/ EmbeddingComposite")
+                print("Ran on the QPU using Qbsolv and dimod's EmbeddingComposite")
```
Comment (bellert) [inline dwave_structural_imbalance_demo/interfaces.py]: The embedding composite is part of `dwave-system`

File: dwave_structural_imbalance_demo/interfaces.py (hunk 1 above)
Comment (JoelPasvolsky) [reply dwave_structural_imbalance_demo/interfaces.py]: Fixed, thank you for catching that!

File: dwave_structural_imbalance_demo/interfaces.py (hunk 2 above)
Comment (JoelPasvolsky) [reply dwave_structural_imbalance_demo/interfaces.py]: Fixed

**Pull request merged**